### DISCUSSION TITLE
How to use custom locale in nextjs13 internationalisation

### DISCUSSION THREAD

**Discussion by ghost** [i18next/next-i18next#2200] [General]
@adrai 
I was following this https://github.com/i18next/next-13-app-dir-i18next-example-ts example, when using locales
'/zh-Hant-HK' and '/zh-Hans-HK' its working fine but we have specific requirement to have routes as '/zh-hant-hk', '/zh-hans-hk', '/ja-jp' and 'ko-kr', is there any way we could make that?

**Comment by adrai** [comment]
Try to set lowerCaseLng to true: https://www.i18next.com/overview/configuration-options#languages-namespaces-resources

**Reply by ghost** [reply]
Thanks for your response, it worked with  lowerCaseLng:true,  is there any way we could use completely custom keywords lang ?

**Reply by adrai** [reply]
not really

**Reply by ghost** [reply]
For my understanding, could I get some insights into where the limitation relies on ? because we could achieve custom keywords with completely client side react application, is the limitation from next js?

**Reply by adrai** [reply]
never investigated...
